src/database/config.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv').config({
```

Comment:
Since Babel has already being configured on the project, Please use ES6 syntax. Change all require to import. For example : ```import {User} from 'Models';```